audit/format_json.go

```diff
@@ -8,13 +8,20 @@ import (
 
 // JSONFormatWriter is an AuditFormatWriter implementation that structures data into
 // a JSON format.
-type JSONFormatWriter struct{}
+type JSONFormatWriter struct {
+	Prefix string
+}
 
 func (f *JSONFormatWriter) WriteRequest(w io.Writer, req *AuditRequestEntry) error {
 	if req == nil {
 		return fmt.Errorf("request entry was nil, cannot encode")
 	}
 
+	_, err := w.Write([]byte(f.Prefix))
```

Comment:
Can you guard this (here and elsewhere) with an `if len(f.Prefix) > 0`? Maybe I'm wrong but that seems like less overhead in the common case than casting the prefix to a byte string every time (and then calling write), even if it's empty.